src/main/resources/templates/smallfull/review.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+    <head>
+        <title>Check your answers</title>
+    </head>
+
+    <body th:object="${review}">
+
+        <div id="balanceSheet-main-content" layout:fragment="content" class="two-column-accounts">
+
+            <h1 id="page-title-heading" class="govuk-heading-l">Check your answers</h1>
+
+            <form id="review-form" th:action="@{''}" class="govuk-body" method="post">
+
+                <!-- Start Balance Sheet -->
+
+                <div th:replace="fragments/review/sectionHeader :: sectionHeader (
+                    description = 'Balance Sheet',
+                    identifier= 'balance-sheet',
+                    resource = 'balance-sheet'
+                    )">
+                </div>
+
+                <dl class="app-check-your-answers app-check-your-answers--short">
+
+                    <div class="app-check-your-answers__contents cya-desktop-only">
+                        <dt class="app-check-your-answers__question"></dt>
+                        <dd class="app-check-your-answers__answer govuk-heading-s"
+                            th:if="*{balanceSheet.balanceSheetHeadings.currentPeriodHeading != null}"
+                            th:text='*{balanceSheet.balanceSheetHeadings.currentPeriodHeading}'></dd>
+                        <dd class="app-check-your-answers__answer govuk-heading-s"
+                            th:if="*{balanceSheet.balanceSheetHeadings.previousPeriodHeading != null}"
+                            th:text='*{balanceSheet.balanceSheetHeadings.previousPeriodHeading}'></dd>
+                        <dd class="app-check-your-answers__change"></dd>
+                    </div>
+
+                    <div th:replace="fragments/review/balanceSheetRow :: balanceSheetRow (
+                        description = 'Called up share capital not paid',
+                        identifier = 'called-up-share-capital-not-paid',
+                        currentAmount = *{balanceSheet?.calledUpShareCapitalNotPaid?.currentAmount},
```

Comment:
Shouldn't need the '?' operator following balanceSheet, not that it makes much difference

Reply:
I agree, however I think the single `?` buried in the middle of this string makes it harder to discern—this is only required where a nested field may be `null` (and only for optional fields on the balance sheet such as the called up share capital). I would prefer to use this for all fields that may be null in a string, or none, depending on whether the field is mandatory or optional.